projects/saas-ui/src/app/main/components/add-plan/add-plan.component.html

```diff
@@ -189,25 +189,63 @@ <h3>Plan Configuration</h3>
                   >
                     <div class="feature-details">
                       <span class="feature-name">{{ feature.name }}</span>
+                      <!-- for boolean -->
                       <div class="feature-input">
                         <nb-toggle
                           *ngIf="feature.type === 'boolean'"
                           [formControlName]="feature.key"
                         ></nb-toggle>
+                        <!-- for number -->
                         <input
                           *ngIf="feature.type === 'number'"
                           nbInput
                           type="number"
                           [formControlName]="feature.key"
                           placeholder="{{ feature.name }}"
                         />
+                        <!-- for conditional metadata handling -->
+
+                        <!-- <select
+                          *ngIf="
+                            feature.metadata && feature.metadata.length > 0
+                          "
+                          [formControlName]="feature.key"
+                        >
+                          <option
+                            *ngFor="let option of feature.metadata"
+                            [value]="option"
+                          >
+                            {{ option }}
+                          </option>
+                        </select> -->
+                        <div class="select">
+                          <nb-select
+                            *ngIf="
+                              feature.metadata && feature.metadata.length > 0
+                            "
+                            class="dropdown-wrapper"
+                            placeholder="Select IDP"
```

Comment:
when the handling is dynamic how can the placeholder be specific to Idp only

Reply:
removed